src/Dfe.Spi.GiasAdapter.Infrastructure.InProcMapping/PocoMapping/EstablishmentMapper.cs

```diff
@@ -74,7 +77,7 @@ public EstablishmentMapper(ITranslator translator)
                 ClosingReason = establishment.ReasonEstablishmentClosed?.Code,
                 PhaseOfEducation = establishment.PhaseOfEducation?.Code,
                 FurtherEducationType = establishment.FurtherEducationType?.DisplayName,
-                SixthForm = establishment.OfficialSixthForm.Code,
+                SixthFormStatus = establishment.OfficialSixthForm.Code,
```

Comment:
Probably needs the null check (I missed it last time)

Reply:
Good spot. 👀 Fixed.